packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -30,8 +31,9 @@ const ChecklistCard = ({
           <p className="text-white typo-callout">{description}</p>
           {typeof onRequestClose === 'function' && (
             <CloseButton
-              buttonSize={ButtonSize.Small}
-              className="top-3 right-3 text-white border-white !absolute btn-secondary"
+              size={ButtonSize.Small}
+              variant={ButtonVariant.Secondary}
+              className="top-3 right-3 text-white border-white !absolute"
```

Comment:
We have position added for `!absolute` right?
Also the border overrides and text are needed on this modal?

Reply:
No, the `position` prop was removed in ButtonV2 in favor of using the absolute className from tailwind, as done here.

Reply:
Ah but then we don't need the important right?
We should avoid having both `relative` and `absolute` at the same time.

Reply:
The original [`Button.tsx` defaults to `position = 'relative'`](https://github.com/dailydotdev/apps/blob/main/packages/shared/src/components/buttons/Button.tsx#L83)

I kept the same behavior for ButtonV2, because I didn't know if we already leveraged that somewhere.

Now that I think about it, it's probably safe to remove that. Any styles that we have can be changed while we migrate to ButtonV2 to match existing behavior. That way we can move to `absolute` instead of `!absolute`. WDYT?

Reply:
Not sure what the proposal would be to only rely on classnames?
My fear is we are going to have to add `relative` everywhere then?

Or did you mean to re-use this position prop?

Reply:
From what I can see, we do not apply any of the properties `top, left, right, bottom, inset, z-index` to Buttons by default. Which means, we do not actually need to render the Button in relative positioning. 

So we can treat adding the `relative` prop where needed, based on existing usage. I can do it while I refactor the rest of the usages. It should be fairly uncommon to use `relative` positioning for Button IMO.

So the proposal is:
```
--- a/packages/shared/src/components/buttons/ButtonV2.tsx
+++ b/packages/shared/src/components/buttons/ButtonV2.tsx
@@ -90,7 +90,7 @@ function ButtonComponent<TagName extends AllowedTags>(
       className={classNames(
         `btn inline-flex flex-row items-center justify-center border
         typo-callout font-bold
-        no-underline shadow-none cursor-pointer select-none focus-outline relative
+        no-underline shadow-none cursor-pointer select-none focus-outline

--- a/packages/shared/src/components/checklist/ChecklistCard.tsx
+++ b/packages/shared/src/components/checklist/ChecklistCard.tsx
@@ -33,7 +33,7 @@ const ChecklistCard = ({
             <CloseButton
               size={ButtonSize.Small}
               variant={ButtonVariant.Secondary}
-              className="top-3 right-3 text-white border-white !absolute"
+              className="top-3 right-3 text-white border-white absolute"
               onClick={onRequestClose}
```

Reply:
In some cases relative might have been added only for z-issues (even just adding relative without z-x could be a benefit.
So I'm open to it, but it would mean we need to double check a lot of things that they didn't get broken.

Reply:
Based on my understanding, unless z-index is specified, `relative` by itself does not create a new stacking order: https://developer.mozilla.org/en-US/docs/Web/CSS/position#relative

Reply:
Let's proceed with your suggestion of removing it and removing the important note on absolutes

Reply:
Should be done in the latest commit